nvflare/private/fed/server/server_engine.py

```diff
@@ -837,7 +837,7 @@ def server_shutdown(server, touch_file):
         os.utime(touch_file, None)
 
     try:
-        server.fl_shutdown()
+        server.fl_shutdown(shutdown_period=30.0)
         server.admin_server.stop()
         time.sleep(3.0)
```

Comment:
For these constant 30.0 and 3.0, we might want to make them configurable as @chesterxgchen suggested.

Reply:
But this is just a note, can be done in a different PR

Reply:
will change it in the next PR.